### DISCUSSION TITLE
Can't zoom in or out in iframe

### DISCUSSION THREAD

**Discussion by ty888** [pmndrs/use-gesture#519] [Q&A] [answered]
**Describe the bug**
There is a problem with zoom on iframe.
If there is an iframe in the operable area, it will be invalid ,see sandbox for details.

**Sandbox or Video**
https://codesandbox.io/s/an-niu-lei-xing-antd-4-21-6-forked-t7h1wx?file=/demo.js
**Information:**
 - Use Gesture version: 10.2.15
 - Device: Mac
 - OS: Mac os
 - Browser : chrome 103.0.5060.114

**Checklist:**
  - [x] I've read the [documentation](https://use-gesture.netlify.app/).
  - [x] If this is an issue with drag, I've tried setting `touch-action: none` to the draggable element.

**Comment by dbismut** [comment]
I won't be able to help without a sandbox.

**Comment by ty888** [comment]
> I won't be able to help without a sandbox.

https://codesandbox.io/s/an-niu-lei-xing-antd-4-21-6-forked-t7h1wx?file=/demo.js

**Comment by dbismut** [comment]
Can you please tick the two checkboxes from the issue template and provide details about the hardware / software? As it seems you haven't set touch-action anywhere. I'll reopen when you fix the sandbox.

**Comment by ty888** [comment]
@dbismut help

**Comment by dbismut** [comment]
So the bug occurs on your laptop?

**Comment by ty888** [comment]
> So the bug occurs on your laptop?

yes

**Comment by ty888** [comment]
@dbismut I've tried using  {pointerEvents: "none"} on the iframe, but that blocks me all.

**Comment by dbismut** [comment]
It's touch-action

**Comment by ty888** [comment]
> It's touch-action

So there is no solution for this problem on laptops

**Comment by dbismut** [comment, marked as answer]
This lib doesn't do much magic. It doesn't interfere with events.

This shows a working sandbox with iFrame zoom and container zoom both working.
https://codesandbox.io/s/cant-zoom-in-or-out-in-iframe-516-forked-xi0ogo
